packages/instrumentation-mongoose/package.json

```diff
@@ -41,17 +41,17 @@
         "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
     },
     "peerDependencies": {
-        "@opentelemetry/api": "^1.0.0"
+        "@opentelemetry/api": "^1.3.0"
     },
     "dependencies": {
         "@opentelemetry/core": "^1.0.0",
```

Comment:
`@opentelemetry/core` is bumped in contrib to `^1.8,0` (since it has peer dependency on API which is now bumped to 1.3.0).


I recommend aligning it here as well 🙏

Reply:
Thanks! Upgraded the packages to `^1.8.0` and fixed the versions in README